tests/functional/z_cluster/test_five_mon.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.pytest_customization.marks import (
+    brown_squad,
+    ignore_leftovers,
+    skipif_less_than_five_workers,
+)
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_ocs_version,
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility import prometheus
+
+
+log = logging.getLogger(__name__)
+
+
+def verify_mon_pod_running(pods):
+    """
+    Verify that all five mon pods are in Running state.
+
+    Returns:
+        bool: True if all mon pods are in running state, False otherwise
+
+    """
+    ret = pods.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector="app=rook-ceph-mon",
+        resource_count=5,
+        timeout=660,
+    )
+    log.info(f"Waited for all mon pods to come up and running {ret}")
+    return ret
```

Comment:
There is no existing library file to validate running mon pod? if not could you please move this to library file